.circleci/config.yml

```diff
@@ -27,6 +28,46 @@ commands:
           fingerprints:
             - "9d:db:07:0b:62:81:11:c3:46:21:8e:fa:51:99:80:2d"
 
+  slack-notify-failure-if-any:
+    steps:
+      - slack/notify:
+          event: fail
+          branch_pattern: ^(main|release.*)$
+          custom: |
+            {
+              "blocks": [
+                {
+                  "type": "header",
+                  "text": {
+                    "type": "plain_text",
+                    "text": ":flutter: Maps SDK Flutter :red_circle:",
+                    "emoji": true
```

Comment:
Should we create a slack handle for you, @maios, and I and flag it here?

Reply:
Good point indeed, I created @maps-flutter and added cc to the template.